lib/config/validation.ts

```diff
@@ -269,6 +269,14 @@ export async function validateConfig(
             if (key === 'extends') {
               for (const subval of val) {
                 if (is.string(subval)) {
```

Comment:
do we issue an error for non string presets?

Reply:
yes we give an error in case its an int or something else. it must be a string!